src/components/NavBar.js

```diff
@@ -0,0 +1,29 @@
+import { FaArrowAltCircleLeft, FaMicrophone } from 'react-icons/fa';
+import { SlSettings } from 'react-icons/sl';
+import { Link } from 'react-router-dom';
+import MealList from '../containers/MealList';
+import '../styles/NavBar.css';
+import '../styles/MealList.css';
+
+const NavBar = () => (
+  <nav data-testid="nav" className="nav">
+    <Link className="back" to="/" onClick={() => MealList()}>
+      {' '}
+      <FaArrowAltCircleLeft className="icons" />
+      {' '}
+    </Link>
+    <div className="home-kitchen">
+      <Link className="heading" to="/">
+        <h2>Home</h2>
+      </Link>
+    </div>
+    <div>
+      <Link className="icons1" to="/">
+        <FaMicrophone className="icons1" />
+        <SlSettings className="icons1" />
+      </Link>
+    </div>
+  </nav>
+);
```

Comment:
- [ ] Kindly improve the styling of your navbar in the mobile versions. This is a mobile web app, which means it should be mobile first. The navbar has some kind of space on the right. Kindly fix that.

  ![image](https://user-images.githubusercontent.com/54848363/227271527-59677751-efbc-4e6b-ae0c-321a9cf76e11.png)